src/AnnotationDialog.js

```diff
@@ -329,17 +328,11 @@ class AnnotationDialog extends EventEmitter {
      * @return {void}
      * @protected
      */
-    addSortedAnnotations(annotations) {
+    sortAnnotationsList(annotations) {
```

Comment:
It looks like annotations is an object in some places and an array in others, which could get very confusing. Is it possible to keep the data type consistent?

Reply:
Made a separate commit so it's easier to follow https://github.com/box/box-annotations/pull/217